pp_rework/pp_example.xml

```diff
@@ -0,0 +1,259 @@
+<!-- ETImporter/FTImporter-->
```

Comment:
(actually a note on the file above, pp_rework/RAVEN Post Processor Plugin Design.docx)
docx instead of latex? docx is quite hard to manage in version control.

Reply:
It is simple enough it could be a .md file.